src/main/java/org/opentripplanner/raptor/api/request/PassThroughPoint.java

```diff
@@ -44,10 +46,17 @@ public int hashCode() {
     return Arrays.hashCode(stops);
   }
 
+  /**
+   * We want the {@code toString()} to be as easy to read as possible for value objects, so the
+   * format for this class is:
+   * <p>
+   * {@code "(<NAME>, <LIST_OF_STOP_INDEXES>)"}, for example:
+   * {@code "(PT-Name, 1, 12, 123)"} or without the name {@code "(1, 12, 123)"}
+   */
   @Override
   public String toString() {
     return (
-      (name == null ? "(" : "(name: '" + name + "', ") + "stops: " + Arrays.toString(stops) + ")"
+      "(" + (name == null ? "" : name + ", ") + "stops: " + IntUtils.intArrayToString(stops) + ")"
```

Comment:
In my opinion, removing the single quotes around the name does not make it easier to read, especially if it contains spaces.